app/pdfs/passengers_PDF.rb

```diff
@@ -0,0 +1,34 @@
+require 'prawn/table'
+
+class PassengersPDF < Prawn::Document
+  def initialize(passengers, filters)
+    super(page_layout: :landscape, page_size: "A3" )
+    content_width = bounds.width - 10
+    header(filters)
+    mycooltable(passengers)
+  end
+
+  def mycooltable(passengers)
+    font_size 14
+    passenger_table = [["Name", "Mobility Device ", "Phone", "Expiration Date", "Notes"]]
+
+    passengers.each do |passenger|
+      name = passenger.name || ''
```

Comment:
I don't think you need all of these `|| ''`s.  That is, doesn't the table cell look the same if you give it `nil` instead of `''`?

Reply:
At least as long ago as 0.0.2 ( the oldest version on GitHub, c. 2014) [it was fine to use `nil`](https://github.com/prawnpdf/prawn-table/blob/dc9aa9bc2a85931739f40fbcad51b69ffea0ce62/lib/prawn/table/cell.rb#L164-L165). So... the last thing?